tests/boot/test_mcuboot/sysbuild.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+# Add the mcuboot key file to the secondary swapped app
+
+if(SB_CONFIG_BOOTLOADER_MCUBOOT)
+  set(swapped_app_CONFIG_BOOTLOADER_MCUBOOT y CACHE STRING
```

Comment:
Why not just stick this in the swapped_app/prj.conf file?

Reply:
For `CONFIG_BOOTLOADER_MCUBOOT`, the `prj.conf` file can be used, and I've moved that KConfig there. `CONFIG_MCUBOOT_SIGNATURE_KEY_FILE` does need to be set like this though, to ensure it takes the same value as the main application and MCUBoot get.